.github/ISSUE_TEMPLATE/meeting.md

```diff
@@ -7,22 +7,40 @@ assignees: ''
 
 ---
 
-Just an hour hanging out and talking about DAT stuff.
+<!--
+This is for listed meetings on the https://comm-comm.datproject.org.
+To be listed this issue needs to have two things:
 
-:point_right: **Post a comment in this github issue with any topics you'd like to cover this week**
+1. a **correct** time 
+2. an **assigned** organizer
+
+Change the meeting time below!
+
+Example: 04 Nov 2020, 13:40 Asia/Tokyo
+Valid Timezones: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+-->
 
 :alarm_clock: Time: DD MMM YYYY, hh:mm GMT
 
-:link: https://zoom.us/j/181036238 (Native)
```

Comment:
We could add

<!-- Choose the hangout you'd like to use (or add the one of your choice). Use only one and comment out the others -->

:link: https://zoom.us/j/181036238 (Native)<br/>
:link: https://zoom.us/wc/join/181036238?pwd= (Web)

<!-- :link: https://talky.io/dat-comm-comm (Web) -->

<!-- :link: https://meet.jit.si/dat-comm-comm (Web) -->